### PR TITLE
Change convert 4 d rgb to 3 d rgb

### PR DESCRIPTION
**Pull request by gabe1496**
-Update function assert_outputs_exists
-Verify the scaling
-Change for supporting the conversion 3D to 4D

### CODE REVIEW

File: scripts/scil_convert_rgb.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Converts a RGB image encoded as a 4D image to a RGB image encoded as
```
Comment (frheault) [inline scripts/scil_convert_rgb.py]: Old doc ?

File: scripts/scil_convert_rgb.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                             assert_outputs_exist)
+
+
+def build_args_parser():
+    p = argparse.ArgumentParser(
+        formatter_class=argparse.RawTextHelpFormatter,
+        description=__doc__)
+
+    p.add_argument('in_image',
+                   help='name of input RGB image.\n' +
```
Comment (frheault) [inline scripts/scil_convert_rgb.py]: Old help ?

File: scripts/scil_convert_rgb.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = build_args_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.in_image])
```
Comment (arnaudbore) [inline scripts/scil_convert_rgb.py]: don't need `[args.in_image]` just `args.in_image`

File: scripts/scil_convert_rgb.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    assert_outputs_exist(parser, args, [args.out_image])
```
Comment (arnaudbore) [inline scripts/scil_convert_rgb.py]: same here

File: scripts/scil_convert_rgb.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+import argparse
+
+from dipy.io.utils import decfa, decfa_to_float
+import nibabel as nb
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def build_args_parser():
```
Comment (arnaudbore) [inline scripts/scil_convert_rgb.py]: please rename _build_args_parser for consistancy

**Comment by gabe1496** [comment]
Data to test
https://drive.google.com/open?id=1Sagb4KlMjhUkhYjVrM9_0WSNn_9oeiAe
https://drive.google.com/open?id=1diaJ891fhNlQnfGaoZQGhehi9X_UEsQe

**Pull request merged**